logger.go

```diff
@@ -49,6 +49,7 @@ type Logger struct {
 	addStack  zapcore.LevelEnabler
 
 	callerSkip int
+	onFatal    zapcore.CheckWriteAction // default is WriteThenFatal.
```

Comment:
```suggestion
	onFatal    zapcore.CheckWriteAction // default is WriteThenFatal
```

This isn't a full sentence so we don't need a trailing `.`